docs/references/chrome-extension/add-react-router.mdx

```diff
@@ -29,44 +29,30 @@ description: Learn how to add React Router to your React application using their
   - Wire layouts and routes up with `createMemoryRouter`
 </TutorialHero>
 
-Learn how to add React Router to your application using React Router's new Data API router. This tutorial will cover configuring layouts and setting up protected routes.
+This tutorial demonstrates how to integrate React Router into your Chrome Extension application using the [Data API router](https://reactrouter.com/en/main/routers/picking-a-router#using-v64-data-apis).
 
 <Steps>
   ### Install `react-router-dom`
 
-  React Router's `react-router-dom` is a mature, battle tested routing package for React that gives you many options. As it is the most popular routing option, it will be used for this guide.
+  React Router is a lightweight, fully-featured routing library. To install it in your project, run the following command:
```

Comment:
Just curious -- this has 'lightweight, full-featured' while the Quickstart removed 'feature rich'. What's the deciding factor with this kind of info?

Reply:
It's from their copy: https://github.com/remix-run/react-router

<img width="874" alt="Screenshot 2024-10-21 at 17 22 17" src="https://github.com/user-attachments/assets/754a06b7-9777-4983-8071-a10d16ad81a5">

Reply:
Ah, gotcha.